python/python_direct/python_translate.cpp

```diff
@@ -883,6 +901,126 @@ class PythonTranslator : public OptInConstDispatch {
         {lop->out()});
   }
 
+  // Map GroupedMmaOp to python frontend
+  void handle(const GroupedMmaOp* gmm_op) final {
+    NVF_ERROR(gmm_op != nullptr);
+    TensorView* out_tv = gmm_op->out();
+    visited_vals_.insert(gmm_op->out());
+
+    int64_t out_block_scale_size = 0;
+    PrimDataType out_block_scale_dtype = DataType::BFloat16;
+    bool out_gamma = false;
+
+    TensorView* out_block_scale_tv = gmm_op->outScale();
+    if (out_block_scale_tv != nullptr) {
+      visited_vals_.insert(gmm_op->outScale());
+      auto block_size_extent = out_block_scale_tv->axis(-1)->extent();
+      NVF_CHECK(
+          block_size_extent->isConstInt(),
```

Comment:
:clap: 

naive question. Should we explicitly get the axis from logical domain? i.e. would the translate function supposed to handle a scheduled fusion?